tests/utils/exec.ts

```diff
@@ -16,6 +15,21 @@ import main from '@/bin/polykey';
 import { promise } from '@/utils';
 import * as validationUtils from '@/validation/utils';
 
+type ExecOpts = {
+  env: Record<string, string | undefined>;
+  command?: string | undefined;
+  cwd?: string;
+  shell?: boolean;
+};
+
+const tsConfigPath = path.resolve(
+  path.join(globalThis.projectDir, 'tsconfig.json'),
+);
```

Comment:
This is not necessary anymore. All configuration of `ts-node` is automatically resolved in `tsconfig.json`. So no need to use `-p`... etc.